doc/onboarding-extras.md

```diff
@@ -13,6 +13,7 @@
 | `lib/{crypto,tls,https}` | @nodejs/crypto |
 | `lib/domains` | @misterdjules |
 | `lib/fs`, `src/{fs|file}` | @nodejs/fs |
+| `lib/internal/url` | @nodejs/url |
```

Comment:
yep... `lib/internal/url.js` is the home of the new implementation. Intentionally put there to keep it distinct from the older `lib/url.js`